hhi-app/src/helper.ts

```diff
@@ -0,0 +1,57 @@
+const CLIENT_ID = process.env.CLIENT_ID;
+const API_KEY = process.env.API_KEY;
+
+// Discovery doc URL for APIs used by the quickstart
+const DISCOVERY_DOC = 'https://sheets.googleapis.com/$discovery/rest?version=v4';
+
+// Authorization scopes required by the API; multiple scopes can be
+// included, separated by spaces.
+const SCOPES = 'https://www.googleapis.com/auth/spreadsheets.readonly';
+let gapiInited = false;
+let gisInited = false;
+
+function gapiLoaded() {
+    gapi.load('client', initializeGapiClient);
+  }
+  async function getItems() {
+    if(!gapiInited || !gisInited){
+      return -1;
+    }
+    let response;
+    try {
+      // Fetch first 10 files
+      response = await gapi.client.sheets.spreadsheets.values.get({
+        spreadsheetId: '1TI0WNrGsTKC_lZPXhcwxK4fHsc2-A88GysVtidI75z0',
+        range: 'Stakeholder Submission Form!B:B',
+      });
+    } catch (err) {
+      return -1;
+    }
+    const range = response.result;
+    if (!range || !range.values || range.values.length == 0) {
+      return -1;
+    }
+    // Flatten to string to display
+    return range.values;
+
+  }
+
+  async function initializeGapiClient() {
+    await gapi.client.init({
+      apiKey: API_KEY,
```

Comment:
Is this ok to expose (because it will be exposed in the bundle if this file is actually used)? What is the scope of this API key? If it has permissions to anything but reading from the spreadsheet that you guys are using, you need to move this logic to live in server-side code.

Reply:
Checked this, the Google Sheets API key doesn't have the granularity for read/write, but does have an URL allowlist that we can use. @AidenCohen31 maybe we should consider using a service account with view only access instead of an API Key here

Reply:
give this one read only access then aiden-842@dssd-hhi.iam.gserviceaccount.com

Reply:
Sounds good, I've shared the above with the sheet.

Is it possible to do the same with a service account on the DSSD email? I think that might be better. (Perhaps we rerecreate what you did on the account above).

Or if you wanted to do so directly, I've also created another account named stakeholder-viewer@hhimap.iam.gserviceaccount.com that has been shared on the sheet. I can share the generated key on that one w you if that works

Reply:
Actually nevermind, looked around a bunch and have discovered that the API key is read only, and the sheet must be shared publically. I think this is ok, as it effectively enables the same as a readonly service account, but with the additional flexibility of site allowlisting.

I think we can proceed with the API solution for now to avoid complexity, since that's implemented already

Reply:
@AidenCohen31 Ping on the above, feel free to approve + merge after resolving your conflicts

Reply:
Related link: https://stackoverflow.com/questions/67058774/how-to-add-api-key-to-the-google-spreadsheet